### DISCUSSION TITLE
USDT from Kucoin to Coinbase using Kava EVM cochain.

### DISCUSSION THREAD

**Discussion by valenciag9** [blockscout/blockscout#8971] [Q&A]
Hello, I'm wondering if anyone can help me recover my life savings. I made the mistake of sending USDT from Kucoin to Coinbase using Kava EVM cochain. Three days have passed and I don't see the funds in Coinbase!!!!

https://kavascan.com/tx/0x699e533fba29709000b8ac9e8e7889b800b053cde1f44ccf53adfd4bc8bca3ea

**Comment by sl1depengwyn** [comment]
This question is not related to Blockscout as a blockchain explorer, contact Kava team directly.
Also, please do not spam your question in both discussions and issues, it does not help.
